lib/src/request.dart

```diff
@@ -52,7 +55,7 @@ class RequestMatcher {
   final RequestHandler requestHandler;
 
   /// This is an artificial response body to the request.
-  ResponseBody responseBody;
+  Responsable responseBody;
```

Comment:
Maybe it's better to rename `responseBody` to `responsable` as it does not only store `ResponseBody` instances now.

Reply:
I think responseBody talks more what it is then responsable, "response body" can be error or response itself, I think it is quite logical to have respoonsebody there.